src/services/scriptingservice.cpp

```diff
@@ -288,12 +290,12 @@ void ScriptingService::initComponents() {
     clearCustomStyleSheets();
     _scriptComponents.clear();
     _settingsVariables.clear();
-    QList<Script> scripts = Script::fetchAll();
 
-    Q_FOREACH (Script script, scripts) {
-        if (script.isEnabled()) {
-            initComponent(script);
-        }
+    //fetch enabled only
```

Comment:
...with a space space after `//`, like everywhere else, pleeeease 😁

Reply:
done

Reply:
Thank you 😁